spec/support/lib/topological_inventory.rb

```diff
@@ -0,0 +1,24 @@
+class TopologicalInventory
```

Comment:
@syncrou Is this file also incorrectly named?
We have this https://github.com/ManageIQ/service_portal-api/blob/master/spec/lib/topological_inventory_spec.rb

Reply:
No it isn't, it's a support file that allows us to call the `TopologicalInventory` class with out making a request to real Topological inventory.  This file helped removing Mocking that was happening for any `ServiceOffering` instance. I'm going to pull this class out of the PR and move all these spec changes into their own PR to help clarify what I was doing here.